Frontend/ui-customer/src/Flow.purs

```diff
@@ -375,6 +378,43 @@ chooseLanguageScreenFlow = do
                             enterMobileNumberScreenFlow
     Refresh state -> chooseLanguageScreenFlow
 
+updateCustomerVersion :: Maybe Version -> Maybe Version -> FlowBT String Unit
+updateCustomerVersion dbClientVersion dbBundleVersion = do
+  let versionName = getValueToLocalStore VERSION_NAME
+      bundle = getValueToLocalStore BUNDLE_VERSION
+      clientVersionArray = (split (Pattern ".") versionName)
+      majorClientVersion = fromMaybe 1 (INT.fromString $ fromMaybe "NA" $ clientVersionArray !! 0)
+      minorClientVersion = fromMaybe 1 (INT.fromString $ fromMaybe "NA" $ clientVersionArray !! 1)
+      patchClientVersion = fromMaybe 1 (INT.fromString $ fromMaybe "NA" $ clientVersionArray !! 2)
+      bundleVersionArray = (split (Pattern ".") bundle)
+      majorBundleVersion = fromMaybe 1 (INT.fromString $ fromMaybe "NA" $ bundleVersionArray !! 0)
+      minorBundleVersion = fromMaybe 1 (INT.fromString $ fromMaybe "NA" $ bundleVersionArray !! 1)
+      patchBundleVersion = fromMaybe 1 (INT.fromString $ fromMaybe "NA" $ bundleVersionArray !! 2)
```

Comment:
if you are using fromMaybe than use current local version as fallback instead of any random value